PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Install JumpCloud Password Manager App.md

```diff
@@ -1,6 +1,6 @@
 #### Name
 
-Windows - Install JumpCloud Password Manager App | v1.0 JCCG
+Windows - Install JumpCloud Password Manager App | v1.1 JCCG
```

Comment:
1.1 is already on the main branch, you'll need to rebase in order to solve for your merge conflicts